quipucords/api/scanjob/model.py

```diff
@@ -149,7 +149,8 @@ def calculate_counts(self, connect_only=False):
 
         :param connect_only: counts should only include
         connection scan results
-        :return: systems_count, systems_scanned, systems_failed
+        :return: systems_count, systems_scanned,
+        systems_failed, systems_unreachable
```

Comment:
To make sure this line is part of the :return: it should be indented. If you check the generated docs you will see this is being considered a new paragraph.

I see this issue happens for other fields in the docstrings too.

Reply:
Yeah, we probably should make a pass at the entire project to fix these issues.  Probably low priority but I agree.